front/src/applications/operationalStudies/views/v2/SimulationResultsV2.tsx

```diff
@@ -154,20 +147,10 @@ const SimulationResultsV2 = ({
           <SimulationWarpedMap collapsed={!showWarpedMap} pathProperties={pathProperties} />
 
           <div className="osrd-simulation-container d-flex flex-grow-1 flex-shrink-1">
-            <div className="chart-container" style={{ height: `${heightOfSpaceTimeChart}px` }}>
-              <SpaceTimeChartV2
-                infraId={infraId}
-                inputSelectedTrain={selectedTrain}
-                trainIdUsedForProjection={trainIdUsedForProjection}
-                simulation={spaceTimeData}
-                simulationIsPlaying={simulationIsPlaying}
-                initialHeight={heightOfSpaceTimeChart}
-                onSetBaseHeight={setHeightOfSpaceTimeChart}
-                dispatchUpdateSelectedTrainId={dispatchUpdateSelectedTrainId}
-                setTrainResultsToFetch={setTrainResultsToFetch}
-                timeScaleDomain={timeScaleDomain}
-                setTimeScaleDomain={setTimeScaleDomain}
-                setTrainSpaceTimeData={setTrainSpaceTimeData}
+            <div className="chart-container">
+              <Manchette
+                operationalPoints={pathProperties.operationalPoints}
+                projectPathTrainResult={manchetteSpaceTimeData}
```

Comment:
You can keep spaceTimeChart, it does not break anything since you have extra props
```suggestion
                projectPathTrainResult={spaceTimeData}
```

Reply:
Actually it's not the same type, as the component is build to use the original type from editoastApi ProjectPathTrainResult[], plus i prefer to keep it in an useMemo

Reply:
As you wish :+1: